### DISCUSSION TITLE
teslamate html file locations?

### DISCUSSION THREAD

**Discussion by houseofdreams** [teslamate-org/teslamate#2387] [Q&A] [answered]
Where can I find the source files for the web pages that are visible at the :4000 port number? (index page / settings page)

**Comment by cwanja** [comment, marked as answer]
Pretty certain it is all here:
https://github.com/adriankumpf/teslamate/tree/master/lib/teslamate_web/live

**Reply by houseofdreams** [reply]
Thanks. Was looking for a way to get some car values, I can't seem to get MQTT to work, so I'm looking for a workaround
